### PR TITLE
Feature/tao 4355 none response processing

### PR DESCRIPTION
**Pull request by ssipasseuth**
1. Enable selecting the "none" response processing mode for all standard interaction
2. Reactivated in the process the answer state for media, upload and end attempt interaction
3. Fixed some eslint warning

### CODE REVIEW

File: views/js/qtiCreator/widgets/interactions/helpers/answerState.js [hunk 1]
```diff
@@ -17,9 +19,55 @@ define([
         }
     };
 
+    /**
+     * Get available rp templates accoding to interaction type and response processing type
+     *
+     * @todo refactor this : check if
```
Comment (ssipasseuth) [inline views/js/qtiCreator/widgets/interactions/helpers/answerState.js]: nope, I have just moved the whole function around to fix the eslint warning ;)

File: model/qti/ParserFactory.php [hunk 2]
```diff
@@ -1303,14 +1301,15 @@ protected function buildTemplatedrivenResponse(DOMElement $data, $interactions){
             }
         }
 
-        // drop rules that don't have a corresponding response identifier
-        if(count(array_diff($responseIdentifiers, array_keys($rules))) > 0){
+        //all rules must have been previously identified as belong to one interaction
```
Comment (ssipasseuth) [inline model/qti/ParserFactory.php]: indeed

File: model/qti/ParserFactory.php [hunk 3]
```diff
@@ -541,17 +541,22 @@ protected function buildItem(DOMElement $data){
...
+            //no response processing node found: the template for an empty response processing is simply "NONE"
+            $rProcessing = new TemplatesDriven();
+            $rProcessing->setRelatedItem($this->item);
+            foreach($this->item->getInteractions() as $interaction){
+                $rProcessing->setTemplate($interaction->getResponse(), Template::NONE);
+            }
+            $this->item->setResponseProcessing($rProcessing);
         }else{
-            $rpNode = $rpNodes->item(0); //the node should be alone
+            //if there is a response processing not, try parsing it
```
Comment (ssipasseuth) [inline model/qti/ParserFactory.php]: indeed

File: views/js/qtiCreator/widgets/choices/inlineChoice/states/Question.js [hunk 4]
```diff
@@ -5,11 +5,14 @@ define([
...
     'lodash'
 ], function($, stateFactory, QuestionState, formElement, _){
+    'use strict';
 
-    var ChoiceStateQuestion = stateFactory.extend(QuestionState, function(){
+    var ChoiceStateQuestion = stateFactory.extend(QuestionState, function initStateQuestion(){
         this.buildEditor();
-    }, function(){
+        console.log('IN');
+    }, function exitStateQuestion(){
```
Comment (no-chris) [inline views/js/qtiCreator/widgets/choices/inlineChoice/states/Question.js]: remaining console.log

File: views/js/qtiCreator/widgets/choices/inlineChoice/states/Question.js [hunk 5]
```diff
@@ -5,11 +5,14 @@ define([
...
         this.destroyEditor();
+        console.log('OUT');
```
Comment (no-chris) [inline views/js/qtiCreator/widgets/choices/inlineChoice/states/Question.js]: also here

**Pull request merged**